dashboard/app/models/concerns/user_permission_grantee.rb

```diff
@@ -52,6 +52,11 @@ def revoke_all_permissions
     end
   end
 
+  # Overrides generated permission helper, above.
+  def ai_api_access?
+    admin? || permission?(UserPermission::AI_API_ACCESS)
```

Comment:
What kind of admin is this supposed to give permissions to? Aren't admins unable to see labs (and thus wouldn't be able to use these apis anyway?) Sorry if I'm missing something, really don't know what's happening in the AI world yet :)

Reply:
That makes sense since this is a permission for access on /allthethings AI Chat levels which we'll be adding soon. Will remove the override. Thanks!